### DISCUSSION TITLE
Notion plugin+add text to database

### DISCUSSION THREAD

**Discussion by nickcbet** [Flow-Launcher/Flow.Launcher#778] [Ideas]
Here is a tutorial i followed for iOS shortcuts which allowed me to use notions api to do a json to the database
https://www.youtube.com/watch?v=sUp8pogYE2Q&t=309s

Basically theres the prompt (flow launcher), then three headers and send file (prompt contents) 
it looks like this 

URL to target: https://api.notion.com/v1/pages
Headers for URL call (remember to set to POST):
Authorization : Bearer INTEGREATIONKEY
Notion-Version: 2021-05-13
Content-Type : application/json

JSON
{"parent": { "database_id": "ID" },
"properties": {"Name" : {"title"    : [{"text": {"content": "Lorem"}}] }}}
The “Lorem” part is what is added to the base “Name” is the actual name of the Title properties in the database.

any framework that is similar that i can edit to make this happen? or is this simple to program myself if i take some time to view some tutorials?

**Comment by jjw24** [comment]
this would be a post request to Notion's db api, which language are you thinking of writing this in?
